BlocQuery/BlocQuery/QuestionDetailViewController.m

```diff
@@ -0,0 +1,125 @@
+//
+//  QuestionDetailViewController.m
+//  BlocQuery
+//
+//  Created by McCay Barnes on 10/26/15.
+//  Copyright © 2015 McCay Barnes. All rights reserved.
+//
+
+#import "QuestionDetailViewController.h"
+
+@interface QuestionDetailViewController () <UITableViewDataSource, UITableViewDelegate>
+
+@property (strong, nonatomic) NSMutableArray *answers;
+@property (weak, nonatomic) IBOutlet UIView *addAnswerView;
+@property (weak, nonatomic) IBOutlet UITextView *answerTextField;
+@property (weak, nonatomic) IBOutlet UITableView *answerTableView;
+@property (strong, nonatomic) NSArray *questions;
+@property (strong, nonatomic) NSString *addedAnswer;
+
+@end
+
+@implementation QuestionDetailViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.addAnswerView.alpha = 0.0;
+    
+    PFQuery *answerQuery = [PFQuery queryWithClassName:@"Question"];
+    [answerQuery whereKey:@"questionText" equalTo:self.selectedQuestion];
+    [answerQuery findObjectsInBackgroundWithBlock:^(NSArray * answers, NSError * _Nullable error) {
+        self.answers = [answers mutableCopy];
+        [self.answerTableView reloadData];
+    }];
```

Comment:
You can include child items in the query itself with `includeKey`. Give that a try.